### ISSUE TITLE
Very long match replay desyncs after first playback.

### ISSUE THREAD

**Issue by GlitchyPSIX** [tetrio/issues#886]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this issue is not already reported.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am using a **supported browser** or the Desktop Client.
- [X] I am not using anything that **modifies game code** (e.g. TETR.IO PLUS).
- [X] I am certain this is a **bug**, not a feature request.

### OS and Version

Windows 10 Home

### Browser and version

TETR.IO Desktop (UA: tetrio-desktop/8.0.0) & Browser on Opera 82.0.4227.58, TETR.IO Build e30RXHWCR

### Describe the bug/issue

After replaying a very long (5 minutes in provided replay) match from a replay, all subsequent replays of that particular match will desynchronize with their actual result unless the replay is reloaded. 

Since it is a long replay, no tests have been done when replaying at speeds other than 5x and 10x. However, this is also replicable by skipping.

In the replay I provide, I win the 5th match. However, replaying it again after shows I wildly misdrop and lose instead, before the replay ends.

Saving a replay after it's damaged will affect the saved replay permanently.

### Reproducing

1. Load a replay with a long match. I have one for your convenience: ID [R:PSIcqM2B0](https://tetr.io/#R:PSIcqM2B0) (Alternatively: [uploaded to my website](glitchypsi.xyz/glitchypsi/tetrioreplays/bug_safe.ttrm))
1.1. (Optional: save the replay file now!)
2. Play the **5th match** once, normally, by skipping or at a high speed.
3. Play the **5th match** again.
4. (optional, if 1.1 was completed) Save replay file with a different name after step 3.
5. (required if 4. was completed) Open second replay file and play the offending match.

### Expected behavior

The match replayed is consistent throughout all replays.

If step 4 was completed, additionally, the match replayed would be as originally seen, instead of the damaged version.

### Screenshots

https://user-images.githubusercontent.com/11571952/151828970-30add2f0-bf57-48e1-8e05-cf0902129f40.mp4

### Console logs

None found.

### Additional context

Important to mention that while I use TETR.IO PLUS in Desktop, this is fully replicable in Browser without it. The recording in Screenshots is infact a recording in the browser specified. It appears the rest of the matches in this replay are OK.

**Comment by GlitchyPSIX** [comment]
Hmmm. Has this ever happened before?

**Comment by ZaptorZap** [comment]
Yeah, it has. There's about [128 closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue+label%3Astale+is%3Aclosed) with the stale tag—so I'm assuming that means 128 of them got closed by this bot.

**Comment by GlitchyPSIX** [comment]
Golly.

**Comment by p1nkl0bst3r** [comment]
The desync happens because after first playback the garbagemultiplier value of the winner is somehow copied to the options of the round. Because the cause is the garbagemultiplier, the issue can occur with any tetra league matches longer than 3 minutes or 10800 frames. This also happens in custom games if garbage margin time increased? 
see this custom game here: [replayissue.ttrm](https://github.com/tetrio/issues/files/9189595/replayissue.ttrm.txt)

this is the difference in the replay files from a tetra league match:

Normal replay file:
![image](https://user-images.githubusercontent.com/45999708/180998929-a60e6f6d-2b61-4f23-8cf9-2773c364edf4.png)

Bugged replay file:
![image](https://user-images.githubusercontent.com/45999708/180998688-4e2586d6-8152-43be-b09a-3943afcf6e27.png)

**Comment by GlitchyPSIX** [comment]
That other issue got closed. Was it solved?

**Comment by p1nkl0bst3r** [comment]
No.
But the issue was not really descriptive anyway, i had already brought the issue to light a couple week prior on the issue report channel but forgot to create and issue for it here.

**Comment by p1nkl0bst3r** [comment]
Issue still present in version 6.3.1

**Comment by GlitchyPSIX** [comment]
the real question is if it has been fixed

**Comment by GlitchyPSIX** [comment]
COME ON

**Comment by lts20050703** [comment]
bruh
